tests/client/client.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package traffic
+package client
```

Comment:
why moving the client out of robustness? Is it useful for other tests?

Reply:
Ups, I meant to move it to tests/robustness/client